apps/prairielearn/src/components/QuestionContainer.html.ts

```diff
@@ -617,11 +617,13 @@ function QuestionPanel({
   return html`
     <div class="card mb-4 question-block">
       <div class="card-header bg-primary text-white d-flex align-items-center">
-        ${QuestionTitle({
-          questionContext,
-          question,
-          questionNumber: instance_question_info?.question_number,
-        })}
+        <h1 class="h6 font-weight-normal mb-0">
+          ${QuestionTitle({
+            questionContext,
+            question,
+            questionNumber: instance_question_info?.question_number,
+          })}
+        </h1>
```

Comment:
For pages where `QuestionContainer` was being used, it made sense that this would be the the `<h1>` heading for those pages so I updated here instead of adding to the page html.

Reply:
I think we could potentially make the question page much nicer for screen readers by adding a lot more navigational landmarks, e.g. separate headings for the question panel, correct answer panel, all submission panels, and so on. Let's save this for another PR though so we can discuss and review that in isolation. Would you like to make an issue for this?

Reply:
That is a great idea. I have now opened issue #10446 to track this.